malasakit-django/pcari/views.py

```diff
@@ -110,8 +107,14 @@ def make_quantitative_question_ratings(respondent, responses):
 def make_comments(respondent, responses):
     for question_id, message in responses.get('comments', {}).iteritems():
         question = QualitativeQuestion(id=int(question_id))
-        yield Comment(respondent=respondent, question=question,
-                      language=respondent.language, message=message)
+
+        # Replaces empty messages with None so they can show up as placeholders in admin 
+        if message == "" or message.isspace():
+            yield Comment(respondent=respondent, question=question,
+                          language=respondent.language, message=None)
+        else:
+            yield Comment(respondent=respondent, question=question,
+                          language=respondent.language, message=message)
```

Comment:
If I can somehow maintain that functionality while not needing `null=True`, that would be great. Last resort would be to manually set all empty responses to `--empty response--` before they are even inserted into the database.